.github/workflows/builds.yml

```diff
@@ -1805,6 +1805,7 @@ jobs:
           CC: "clang-15"
           CXX: "clang++-15"
           RUSTFLAGS: "-C instrument-coverage"
+          SURICATA_LUA_SYS_CFLAGS: "-fsanitize=address"
```

Comment:
Why do we need this in this CI build ?

Reply:
This is a fuzz build? Should it not be there?

Reply:
It is ok, but here we are more interested about the coverage (of the fuzz targets) than the actual fuzzing ;-)

Reply:
Ok, lets leave it if its not going to be issue.  Helps me know that it was actually picked up (I did remove the -vvvvv though to verify).